internal/events/eventstream.go

```diff
@@ -164,7 +164,7 @@ func (es *eventStream) initAction(startedState *startedStreamState) {
 	case apitypes.EventStreamTypeWebhook:
 		startedState.action = newWebhookAction(ctx, es.spec.Webhook).attemptBatch
 	case apitypes.EventStreamTypeWebSocket:
-		startedState.action = newWebSocketAction(es.wsChannels, es.spec.WebSocket, *es.spec.Name).attemptBatch
+		startedState.action = newWebSocketAction(es.wsChannels, es.spec.WebSocket, *es.spec.WebSocket.Topic).attemptBatch
```

Comment:
What is the migration impact of this one? Do we need to edit the Topic of existing pre-v1.1 event streams on startup (before this) to ensure migration works with EthConnect v1.0 environments?